ticdc/ticdc-overview.md

```diff
@@ -41,13 +45,29 @@ Currently, the TiCDC sink component supports replicating data to the following d
 - Databases compatible with MySQL protocol. The sink component provides the final consistency support.
 - Kafka based on the TiCDC Open Protocol. The sink component ensures the row-level order, final consistency or strict transactional consistency.
 
-### Black and white table lists
-
-You can write blacklist and whitelist filtering rules to filter or only replicate all changed data in certain databases or tables. The filtering rules are similar to those of MySQL such as `replication-rules-db` or `replication-rules-table`.
-
 ## Restrictions
 
 To replicate data to TiDB or MySQL, you must ensure that the following requirements are satisfied to guarantee data correctness:
 
 - The table to be replicated has the primary key or a unique index.
 - If the table to be replicated only has unique indexes, each column of at least one unique index is explicitly defined in the table schema as `NOT NULL`.
+
+### Unsupported scenarios
+
+Currently, The following scenarios are not supported:
+
+- The replication of partition tables.
+- The TiKV cluster that uses RawKV alone.
+- The [new framework for collations](/character-set-and-collation.md#new-framework-for-collations) in TiDB v4.0. Before you enable this feature, make sure that the downstream cluster is the TiDB cluster with the same collation as those in the upstream; otherwise, inconsistent collations lead to the issue that the data cannot be located.
+- [The DDL operation `CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md) and the [SEQUENCE function](/sql-statements/sql-statement-create-sequence.md#sequence-function) in TiDB v4.0. When the upstream TiDB uses SEQUENCE, TiCDC ignores SEQUENCE DDL operations/functions performed upstream. However, DML operations using SEQUENCE functions can be correctly replicated.
+- The [TiKV Hibernate Region](https://github.com/tikv/tikv/blob/master/docs/reference/configuration/raftstore-config.md#hibernate-region). TiCDC prevents the Region from entering the hibernated state.
+- The scheduling of existing replication tables to new TiCDC nodes, after the capacity of the TiCDC cluster is scaled out.
+- The replication of Black and white table lists.
+
+## Manage TiCDC Cluster and Replication Tasks
+
+For details, see [Manage TiCDC Cluster and Replication Tasks](/ticdc/manage-ticdc.md).
+
```

Comment:
I can't find the troubleshooting file in the repo. Is it a P2 doc? 
If so, ignore this comment. We can add this link after troubleshoot-ticdc.md is ready.
```suggestion

## TiCDC troubleshooting

For the common questions and corresponding solutions during the use of TiCDC, refer to [TiCDC Troubleshooting](/ticdc/troubleshoot-ticdc.md)

```